karate-tests/src/test/java/mocks/sandbox/get-patient-search.js

```diff
@@ -21,7 +21,7 @@ const JACKIE_JONES_SEARCHSET = context.read('classpath:mocks/stubs/searchRespons
 const RODNEY_GODSOE_SEARCHSET = context.read('classpath:mocks/stubs/searchResponses/rodney_godsoe_searchset.json')
 const MARTHA_MASSAM_SEARCHSET = context.read('classpath:mocks/stubs/searchResponses/martha_massam_searchset.json')
 const YOUDS_SEARCHSET = context.read('classpath:mocks/stubs/searchResponses/youds_searchset.json')
-const BILL_GARTON_SEARCHSET = context.read('classpath:mocks/stubs/searchResponses/bill_garton_searchset.json')
+const JOE_BLOGS_HISOTRIC_NAME_SEARCHSET = context.read('classpath:mocks/stubs/searchResponses/bill_garton_searchset.json')
```

Comment:
File path still points to bill_garton_searchset.json

Reply:
It still needs to point to that record as Bill Garton is the current name / what it's returning. It's matching on Joe Blogs which is the historic name.
Or are you suggesting renaming the file as well to something like joe_blogs_historic_searchset.json?